web/src/App.css

```diff
@@ -1,38 +1,41 @@
-.App {
-  text-align: center;
+* {
+  box-sizing: border-box;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.grid-container {
+  display: grid;
+  grid-template-areas:
+        'nav nav nav nav nav nav'
+        'task-tree task-tree task-tree task-tree task-tree task-list'
```

Comment:
what is this?

Reply:
https://developer.mozilla.org/de/docs/Web/CSS/grid-template-areas